modules/kmm-setting-soft-dependencies-between-kernel-modules.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-setting-soft-dependencies-between-kernel-modules_{context}"]
+= Set soft dependencies between kernel modules
+
+Some configurations require that several kernel modules be loaded in a specific order to work properly, even though the modules do not directly depend on each other through symbols.
```

Comment:
Do we need to define what a configuration is here?

Reply:
Do we need _through symbols_? Might be just me, but I don't know what that means.
```suggestion
Some configurations require that several kernel modules be loaded in a specific order, even though the modules do not directly depend on each other.
```

Reply:
@mburke5678 I had actually changed that from "setup". It makes more sense and the readers know what it means.